web/modules/custom/joinup_core/joinup_core.module

```diff
@@ -1038,12 +1040,8 @@ function joinup_core_preprocess_page(&$variables) {
   if (!empty($context['og'])) {
     $group = $context['og']->getContextValue();
 
-    if (
-      $group &&
-      $group->getEntityTypeId() === 'rdf_entity' &&
-      $group->bundle() === 'collection'
-    ) {
-      $variables['attributes']['data-drupal-collection-context'] = $group->id();
+    if ($group && $group->getEntityTypeId() === 'rdf_entity') {
+      $variables['attributes']['data-drupal-group-context'] = $group->id();
```

Comment:
I can confirm that just updating the selector to `data-drupal-group-context` fixes the issue.

Reply:
Great :D thanks for the catch! To be honest, I was reassured that everything is fine since the tests were green. Since we write tests for everything, I thought that this is already searched.